resources/js/app.js

```diff
@@ -534,3 +534,17 @@ function Kanban() {
         }
     }
 }
+
+$(function(){
```

Comment:
What do mean by template code? I thought these files where intended to hold all js code. When I tried to put it into the form it didn't work at all.